Automattic/pocket-casts-ios
# Show all sorting options for episode

| 📘 Part of: # |  
|:---:|

This PR enables showing all available options for episode sorting by adding TitleAtoZ and TitleZtoA.
This will match the sort options in Android and Web

<img src="https://github.com/user-attachments/assets/2ce510b4-1680-40e7-a07c-2c308f664e2f" width=320>


## To test

1. Start the app
2. Open a Podcast
3. Tap on the ... button
4. Select the Sort row
5. Check if you see the options to sort for TitleAtoZ and TitleZtoA
6. Select each of the sorts and see if they sort correctly

## Checklist
